x-pack/plugins/fleet/server/types/models/agent_policy.ts

```diff
@@ -64,7 +64,18 @@ export const AgentPolicyBaseSchema = {
       ])
     )
   ),
-  keep_monitoring_alive: schema.maybe(schema.boolean({ defaultValue: false })),
+  keep_monitoring_alive: schema.maybe(
+    schema.oneOf([
+      schema.literal(null),
+      schema.boolean({
+        defaultValue: false,
+        meta: {
+          description:
+            ' When set to true, monitoring will be enabled but logs/metrics collection will be disabled',
```

Comment:
Nit: There's an unneeded space at the beginning

Reply:
removed